c3r-sdk-core/build.gradle

```diff
@@ -224,14 +224,19 @@ publishing {
                     connection = 'scm:git:ssh://git@github.com/aws/c3r.git'
                     developerConnection = 'scm:git:ssh://git@github.com/aws/c3r.git'
                 }
-
                 licenses {
                     license {
                         name = 'The Apache License, Version 2.0'
                         url = 'https://www.apache.org/licenses/LICENSE-2.0.txt'
                         distribution = 'repo'
                     }
                 }
+                developers {
+                    developer {
+                        id = "amazonwebservices"
```

Comment:
Random question, is this standard? Or should it be `amazonaws` for the ID like the package name?

Reply:
`amazonwebservices` is the developer id that is used in these kind of packages (scroll down to "Developers" section), e.g.:
+ https://mvnrepository.com/artifact/com.amazonaws/aws-dynamodb-encryption-java/2.0.3
+ https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-core/1.12.393
+ https://mvnrepository.com/artifact/com.amazonaws/amazon-kinesis-client/1.14.9